.github/workflows/ci-check.yml

```diff
@@ -36,6 +36,7 @@ jobs:
         run: exit $(git status --porcelain | wc -l)
   frontend-tests:
     runs-on: ubuntu-latest
+    concurrency: cypress-tests
```

Comment:
Is `cypress-tests` just an arbitrary environment name?

Reply:
Yep